### DISCUSSION TITLE
Images not loading lazily?

### DISCUSSION THREAD

**Discussion by FarazPatankar** [ascorbic/unpic-img#664] [Q&A] [answered]
Hey y'all,

Tried out the library today and it seems images are ALL eagerly loaded even though the default is supposed to be `lazy`? The `img` element also has `loading="lazy"` on it and my code is pretty basic:
```tsx
<Image
  src={c.image + "/low.webp"}
  alt={c.name}
  layout="constrained"
  height={337}
  width={245}
/>
```

You can see it live [here](https://farazpatankar.com/p/scarlet-and-violet-151).

**Comment by ascorbic** [comment]
Which framework is this?

**Reply by FarazPatankar** [reply]
I am using Remix. This is [open source](https://github.com/FarazPatankar/fp/blob/main/app/components/custom/ScarletAndViolet151.tsx#L77) as well.

**Comment by ascorbic** [comment, marked as answer]
It looks like the problem is that the images are all onscreen when the  page first loads. You can see this in the first flash of loading, where the badges are all visible. Because they're onscreen, they all start to load. I've not looked closely at the CSS, but it's going to be related to the fact that constrained images will scale down if their wrapper is smaller than their max width.

**Reply by FarazPatankar** [reply]
Yep, you're right. Adding a `min-height` seems to have fixed it. Thanks! 😄
